docs_outdoornav_user_manual/web_user_interface/ui_autonomous_mode.mdx

```diff
@@ -259,39 +268,45 @@ class CustomTask(CustomTaskBase):
       self._as.publish_feedback(self._feedback)
 
     return True
-```
 
-This sample custom task iterates through the two available data variables and outputs them to the feedback topic. If neither of the 
-variables have any data in them the task is aborted.
+def __create_task__():
+	return InputLooperTask()

```

Comment:
It might be worth checking with @civerachb-cpr or @mhosmar-cpr here. I think that this probably shouldn't be needed and that there should be an existing way in python to load all the modules in a directory, but they might know more.

Reply:
removed